### PR TITLE
Add SBRPs for vstest

### PR DESCRIPTION
**Pull request by NikolaMilosavljevic**
SBRPs required for `vstest` per-repo prebuilts work: https://github.com/microsoft/vstest/issues/4405

```
Microsoft.Build.Framework.16.0.461
Microsoft.Build.Utilities.Core.16.0.461
Microsoft.Extensions.FileSystemGlobbing.2.0.0
NuGet.Frameworks.5.11.0
```

### CODE REVIEW

File: src/referencePackages/src/microsoft.extensions.filesystemglobbing/2.0.0/Microsoft.Extensions.FileSystemGlobbing.2.0.0.csproj
Comment (MichaelSimons) [inline src/referencePackages/src/microsoft.extensions.filesystemglobbing/2.0.0/Microsoft.Extensions.FileSystemGlobbing.2.0.0.csproj:1]: This is an incredibly old version - 2017.  Is it feasible to upgrade to a newer version possible on already in SBRP.

File: src/referencePackages/src/microsoft.build.framework/16.0.461/Microsoft.Build.Framework.16.0.461.csproj
Comment (MichaelSimons) [inline src/referencePackages/src/microsoft.build.framework/16.0.461/Microsoft.Build.Framework.16.0.461.csproj:1]: This is a pretty old version - 2019.  What is the feasibility of getting them to upgrade to a newer version?  Preferrably one in SBRP.

**Review by MichaelSimons — COMMENTED** [review]
Would be nice to see vstest upgrade to some newer versions.  If that is not feasible I am alright with the changes.

**Comment by NikolaMilosavljevic** [comment]
> Would be nice to see vstest upgrade to some newer versions. If that is not feasible I am alright with the changes.

I'm going to inquire, but I'm hesitant to further delay `vstest` onboarding to per-repo prebuilts verification. Would it help if we had issues in `vstest` to request updates to newer versions?

Otoh, we are still adding older versions for some other repos, i.e. https://github.com/dotnet/source-build-reference-packages/pull/602

**Comment by MichaelSimons** [comment]
I agree we should not delay prebuilt detection.  We should log a follow-up issue for them to upgrade.

**Review by MichaelSimons — APPROVED** [review]

**Comment by NikolaMilosavljevic** [comment]
Created a tracking issue for old packages: https://github.com/microsoft/vstest/issues/4418

**Pull request merged**